src/QueryFilterHelpers.ts

```diff
@@ -207,38 +217,55 @@ export function interpretExpression(
   parameters: any,
   patient: R4.IPatient
 ): AnyFilter {
+  let returnFilter: AnyFilter = {
+    type: ''
```

Comment:
I highly suggest making this default to be an UnknownFilter.
```suggestion
    type: 'unknown'
```